homework7/hw7_task1/task_find_occurrences/find_occurrences.py

```diff
@@ -0,0 +1,33 @@
+"""
+Given a dictionary (tree), that can contains multiple nested structures.
+Write a function, that takes element and finds the number of occurrences
+of this element in the tree.
+Tree can only contains basic structures like:
+    str, list, tuple, dict, set, int, bool
+"""
+from typing import Any, Iterable, Iterator, Mapping
+
+
+def recursively_read(tree: Iterable, element, occurrences: list):
```

Comment:
`occurrences` usually called `accumulator`  But in that case do you really need it?

Reply:
Do you mean it is not a good idea to use collection (list) for store and change just one element? Another variant is to use occurances = 0 as a global variable, but I heard somewhere that global variables are not a good practice because another person in project can use this variable in his aims and it can cause some conflict. Is it right?

Reply:
There is another way.  It is almost implemented, just a couple of small changes required.

Reply:
Done! It was always a problem for me how to deal with value, how to store it and increase after new func call. And now I understand how close I was to this step:D